### ISSUE TITLE
Gnome Not Working

### ISSUE THREAD

**Issue by RaufNawaz** [yeyushengfan258/We10X-icon-theme#15]
Upon applying this theme most of my application icons disappeared, and gnome tweaks as well as terminal and several other applications that use its icons stopped working. Even parts of my shell are not visible.

**Comment by yeyushengfan258** [comment]
Is there not enough storage space? This is the first accident I heard, and I don't know how to deal with it yet. 
Sorry.

**Comment by RaufNawaz** [comment]
Umm, i have around 100 gb available so thats probably not the issue. I can send in a screenshot if that would help.

**Comment by microhobby** [comment]
Yeah same here, I was using old versions without issues. I also tried the Win11 but a lot of icons are missed.

**Comment by yeyushengfan258** [comment]
I also have a lot of storage space, but the cache space is different, and sometimes some files fail to install because of insufficient cache space.
These are some of the reasons I currently know.

**Comment by yeyushengfan258** [comment]
I have updated a batch of link icons for We10X, so some icons will disappear, but because the new link icons are more than double the original, I can only add them separately through your needs icons.

**Comment by yeyushengfan258** [comment]
The Win11 icon is not perfect yet, only some basic system is added, which will be improved in the later stage, and you can also mention it here if you need it, and I will add it in the update.
